lane_detection:
    ros__parameters:
        debug_view:                           false
        frame_id:
            lane_geometry:                    'laneGeometryFrameId'
            lane_parameters:                  'laneParametersFrameId'
        lane_color:                           [50,234,157] # location:(537, 382)
        m_per_pix:
            x:                                0.010278
            y:                                0.033333
        qos_length:                           10
        roi:
            height:                           202
            width1:                           66
            width2:                           405
            x1_offset:                        327
            x2_offset:                        157
            y_offset:                         278
        topic:
            lane_geometry:                    'lane/geometry'
            lane_parameters:                  'lane/parameters'
            segmentation:                     'carla/ego_vehicle/semantic_segmentation_front/image'
            